mlrun/feature_store/api.py

```diff
@@ -582,3 +582,19 @@ def set_task_params(
         featureset.set_targets()
     featureset.save()
     return source, parameters
+
+
+def delete_feature_set(name, project=""):
```

Comment:
@urihoenig what about tag/ver?

Reply:
@yaronha we're currently don't support deleting a specific version - only the whole feature set (with all versions)